build.gradle

```diff
@@ -1,32 +1,48 @@
+configurations {
+    warLibs
+}
 buildscript {
     repositories {
         mavenLocal()
         jcenter()
         maven {
             url "https://plugins.gradle.org/m2/"
         }
-        maven { 
-            url artifactoryURL 
+        maven {
+            url artifactoryURL
             credentials {
                 username = artifactoryUser
                 password = artifactoryPassword
             }
         }
     }
+
+
     dependencies {
         classpath 'org.ajoberstar:gradle-git:1.4.2'
         classpath 'net.ltgt.gradle:gradle-errorprone-plugin:0.0.13'
         classpath 'com.github.jengelman.gradle.plugins:shadow:2.0.1'
     }
+
+}
+
+dependencies {
+    warLibs 'org.ajoberstar:gradle-git:1.4.2'
+    warLibs 'net.ltgt.gradle:gradle-errorprone-plugin:0.0.13'
+    warLibs 'com.github.jengelman.gradle.plugins:shadow:2.0.1'
```

Comment:
You're adding these to the war libs? These are meant for buildscript execution only and should not be part of anything we ship.

Reply:
You are correct. These should not be added.